libs/block-content/website/src/lib/break/break-block.tsx

```diff
@@ -5,42 +5,36 @@ import {Block, LinkPageBreakBlock as LinkPageBreakBlockType} from '@wepublish/we
 export const isBreakBlock = (block: Block): block is LinkPageBreakBlockType =>
   block.__typename === 'LinkPageBreakBlock'
 
-export const BreakBlockWrapper = styled('div')<{reverse?: boolean}>`
+export const BreakBlockWrapper = styled('div')`
   display: grid;
   gap: ${({theme}) => theme.spacing(2)};
-  grid-template-columns: 100%;
   justify-content: center;
 
   ${({theme}) => theme.breakpoints.up('md')} {
-    grid-auto-flow: ${props => props.reverse && 'column'};
-    gap: ${({theme}) => theme.spacing(20)};
-    grid-template-columns: repeat(2, 1fr);
-  }
-
-  ${({theme}) => theme.breakpoints.up('lg')} {
-    gap: ${({theme}) => theme.spacing(30)};
+    grid-template-columns: 1fr 2fr;
   }
 `
 
 export const BreakBlockSegment = styled('div')<{reverse?: boolean}>`
   display: grid;
-  width: 100%;
   align-items: center;
-  ${({theme}) => theme.breakpoints.up('md')} {
-    grid-column-end: ${props => props.reverse && '-1'};
-  }
-`
+  gap: ${({theme}) => theme.spacing(2)};
 
-const headingStyles = css`
-  margin-bottom: 24px;
+  ${({theme, reverse}) =>
+    reverse &&
+    css`
+      ${theme.breakpoints.up('md')} {
+        order: 1;
```

Comment:
Reversing can be done by using order

Reply:
I couldn't make it work for some reason, and how I did it I found somewhere on stack overflow. And actually many people wrote that the way I did it is the only way 😆